core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -1004,6 +1014,43 @@ def __retrieve_atp_rcudbinfo(self, rcu_db_info, checkAdminPwd=False):
 
         return tns_admin, rcu_database, keystore_pwd, truststore_pwd
 
+    def __retrieve_ssl_rcudbinfo(self, rcu_db_info, checkAdminPwd=False):
+        """
+        Check and return ssl connection info and make sure ssl rcudb info is complete
+        :raises: CreateException: if an error occurs
+        """
+        _method_name = '__retrieve_ssl_rcudbinfo'
+
+        tns_admin = rcu_db_info.get_ssl_tns_admin()
```

Comment:
Is tns_admin required field ?   If yes, we should probably check the truststore field value and the combination of them pointing to an existing file

Reply:
Added thetruststore check to the tnsnames check